docs/src/app/app.native.js

```diff
@@ -1,4 +1,6 @@
-import React, {Component, PropTypes} from 'react-native';
+import React, {Component} from 'react-native';
+
```

Comment:
Minor issue, we have quite some blank lines added by this PR between the imports.